consumer.go

```diff
@@ -454,7 +454,15 @@ func (c *Consumer) rebalance() (map[string][]int32, error) {
 	sarama.Logger.Printf("cluster/consumer %s rebalance\n", c.memberID)
 
 	if err := c.client.RefreshMetadata(); err != nil {
```

Comment:
Could you please move the whole block into a `func (c *Consumer) refreshMetatdata()`?